ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -755,19 +755,22 @@ objects:
           fi
         fi
 
-        if has_variant "ipv6"; then
-          export OPENSHIFT_INSTALL_AZURE_EMULATE_SINGLESTACK_IPV6=true
+        if has_variant "dualstack"; then
+          export OPENSHIFT_INSTALL_EXPERIMENTAL_DUAL_STACK=true
```

Comment:
lol wait how many places are you going to specify this in? :slightly_smiling_face:

Reply:
puff, I went crazy trying to follow the dependencies, it seems there was an "old way" using templates, that the "current way" doesn't use it.
I think that cluster-bot still uses this template